project/BuildCommon.scala

```diff
@@ -1,4 +1,4 @@
-import com.trafficland.augmentsbt._
+import com.trafficland.augmentsbt.versionmanagement.VersionManagementPlugin.autoImport.toVersion
```

Comment:
You moved it so that it can be auto-imported but auto-import won't work in non-sbt files.

That sucks.